ReactiveCocoa/Swift/Property.swift

```diff
@@ -95,6 +95,57 @@ extension MutableProperty: SinkType {
 	}
 }
 
+/// Wraps a `dynamic` property, or one defined in Objective-C, using Key-Value
+/// Coding and Key-Value Observing.
+///
+/// Use this class only as a last resort! `MutableProperty` is generally better
+/// unless KVC/KVO is required by the API you're using (for example,
+/// `NSOperation`).
+@objc public final class DynamicProperty: NSObject, PropertyType {
+	public typealias Value = AnyObject?
```

Comment:
I agree, but if it doesn't crash in RAC, I would make my app crash anyway, by using `as!` when I get a value from the signal. Put another way: how is an app supposed to handle gracefully the KVO subscription sending values of a type you didn't expect? I think most Obj-C apps would crash with "unrecognized selector" due to the implicit cast, cause we rarely do isKindOfClass in those cases...